src/main/g8/README.md

```diff
@@ -89,19 +89,28 @@ Below are the libraries used to provide a broad starting base for this project.
 
 
 
-## [sbt-native-packager](https://github.com/sbt/sbt-native-packager) - JRE-less executable ##
+## [sbt-native-image](https://github.com/scalameta/sbt-native-image) - JRE-less executable ##
 
-`sbt-native-packager` is an sbt plugin that allows sbt to build various distributables.
+`sbt-native-image` is an sbt plugin that allows sbt to build assembled JRE-less executables using GraalVM.  This plugin handles downloading and installation of the related Graal JDK and native-image tool which are required to make the executable binary.  However, these are very particular about their targets - That's where we pick up with docker in this repo, ultimately ending up with a portable, fast, minified docker image with a single binary in it:
+
+1. **sbt + sbt-native-image + GraalVM + native-image** get us from **scala-to-bytecode-to-llvm-to-libc**.
+2. **docker** gets us from **libc-to-kernel**, providing a consistent "target" for GraalVM & native-image.
+
+However, it isn't all roses.  **The native-image & AoT aspect of any project building with Graal is very brittle**, as all classes exposed exclusively via reflection at run-time [must be explicitly provided to Graal](https://www.graalvm.org/reference-manual/native-image/Reflection/), whether determined automatically or manually.  
+
+Do your reading before you decide to push the outputs of native-image to production, and consider the cost of maintaining reflection mappings.
 
 ###### Why do we use it? ######
 
-- GraalVM integration allows us to build small fast-starting binaries.
-- Bake dependencies into binaries.
+- GraalVM integration allows us to build small fast-starting binaries out of Scala that would have otherwise been bytecode.  Throwing the JRE baby out with the bathwater offers several benefits:
+  - No JVM cold start
+  - AoT target-based compilation means no JIT jitters or hotspot hotflashes
+  - Tiny docker images (tens of MB) with the smallest possible base image *(even from [scratch](https://hub.docker.com/_/scratch), if you want)*
```

Comment:
Pretty sure I even had a 6MB image one time, which was pretty freakin' crazy.

Reply:
Yikes. I'd hate to shell into that badboy.